src/components/HomePageContent/HowItWorksSection/Steps.tsx

```diff
@@ -15,13 +15,22 @@ export const Steps: React.FunctionComponent<any> = ({ contentType, stepsDetails
       <div className="flex flex-wrap m-auto lg:px-16 2xl:px-0 items-start lg:w-full 2xl:max-w-7xl">
         {stepsDetails.map((steps: any, index: any) => (
           <>
-            <div key={index} className="flex flex-row flex-none w-6/12 mt-8 justify-center lg:w-auto">
+            <div
+              key={index}
```

Comment:
change the <> to <React.Fragment key={index}> to fix the key warning, and remove the key={index} for line 19

Reply:
done.